extensions-core/kafka-indexing-service/src/test/java/org/apache/druid/indexing/kafka/supervisor/KafkaSupervisorTest.java

```diff
@@ -613,6 +617,80 @@ public void testPartitionIdsUpdates() throws Exception
     Assert.assertFalse(supervisor.isPartitionIdsEmpty());
   }
 
+
+  /**
+   * Regression Test for if always use earliest offset on newly discovered partitions
+   */
+  @Ignore
```

Comment:
This annotation accepts a string value that you can use to add a reason why the test is ignored